cmd/flux/cmd/test.go

```diff
@@ -24,6 +24,40 @@ import (
 	"github.com/spf13/cobra"
 )
 
+var testCommand = &cobra.Command{
+	Use:   "test",
+	Short: "Run flux tests",
+	Long:  "Run flux tests",
+	Run: func(cmd *cobra.Command, args []string) {
+		fluxinit.FluxInit()
+		runFluxTests()
+	},
+}
+
+var testNames []string
+var rootDir string
+var verbosity int
+
+func init() {
+	rootCmd.AddCommand(testCommand)
+	testCommand.Flags().StringVarP(&rootDir, "path", "p", "./stdlib", "The root level directory for all packages.")
```

Comment:
A default that is relative to the repo root seems a bit odd to me. I can see how it will work well for us but is there a better long term solution? 

Also can you set this path via the ENV? I don't remember if cobra gets us that for free or not.

Reply:
It also feels awkward to me, though I wanted to wait until we knew what the other test runner commands in downstream consumers looked like before making a definitive decision there.